src/legacy/core_plugins/timelion/public/vis/timelion_request_handler.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import _ from 'lodash';
+// @ts-ignore
+import { buildEsQuery, getEsQueryConfig, Filter } from '@kbn/es-query';
+// @ts-ignore
+import { timezoneProvider } from 'ui/vis/lib/timezone';
+import { KIBANA_CONTEXT_NAME } from 'src/plugins/data/common/expressions/types';
```

Comment:
@lukeelmers  Are  you ok with that? 

@Avinar-24  Please use relative path for imports

Reply:
I'm not sure the claim is the best one.

Let's compare two versions:
`import { something } from '../../../../expressions/types';`
`import { something } from 'src/plugins/data/common/expressions/types';`

At first glance if someone looked at the piece of code what he would find out?

**In the first** case we only see that something is a part of some types of some expressions.
**In the second** one we see it's definitely types of expressions of the data plugin. That's looks more obvious for those who faces the code first time.

The reason where relative paths kind of `../../` are more suitable is when something is imported inside one plugin. For instance, if I worked with `Timelion` and I did import somewhere like:

`import { something } from '../types'`

I would be sure that `something` is a part of types of the **Timelion**.

@lukeelmers How do you feel with the statement?

Reply:
Currently, using relative imports is the only 100% reliable way to do imports which work in all scenarios (TS, JS, Jest, Mocha tests, etc)

But eventually we are hoping to get to a place where you could use either relative _or_ absolute paths; see #40446 

I agree @Avinar-24, absolute imports can be much clearer. So I'm comfortable with using either; as long as you don't hit CI issues.

One thing I would suggest though is that ideally we would be importing from `data/public` here -- we shouldn't be importing from `common` directly. I'm not sure if we are already exporting this from the top-level of the data plugin though... if we aren't, we can probably wait on #45125 before making this change

Reply:
Paths has been changed and looks pretty now